### PR TITLE
Highlight client only

### PR DESCRIPTION
**Pull request by NorwegianKiwi**
See also comments in PR #140.

### CODE REVIEW

File: src/index-template.ejs [hunk 1]
```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <div id="app"><%= htmlWebpackPlugin.options.appcontent %></div>
+    <script type="text/javascript" src="http://scratchblocks.github.io/js/scratchblocks-v3.1-min.js"></script>
+    <script type="text/javascript" src="http://scratchblocks.github.io/js/translations-all-v3.1-min.js"></script>
```
Comment (arve0) [inline src/index-template.ejs]: This language pack is quite large, and may disappear. Use the Norwegian build in arve0/scratchblocks#no_build

Comment (arve0) [inline src/index-template.ejs]: Or create own language pack, as described here: 

https://github.com/arve0/scratchblocks/tree/master/src/locales

Comment (arve0) [inline src/index-template.ejs]: See the updated #70, which uses a new browser-build which includes the translations.

**Comment by NorwegianKiwi** [comment]
Using PR #70 instead.

**Pull request closed without merging**